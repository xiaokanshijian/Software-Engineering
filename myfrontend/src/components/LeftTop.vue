<template>
  
</template>

<script>
export default {
    props:{
        config1:{}
    },
    watch:{
        config1(){
        }
    }
}
</script>

<style>

</style>