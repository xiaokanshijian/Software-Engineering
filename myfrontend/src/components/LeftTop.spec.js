import { shallowMount } from '@vue/test-utils';
import LeftTop from './LeftTop.vue';





describe('<LeftTop/>', () => {
	const wrapper = shallowMount(LeftTop);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(LeftTop);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

