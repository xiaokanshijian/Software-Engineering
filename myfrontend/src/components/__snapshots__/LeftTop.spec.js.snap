// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeftTop/> snapshot测试 1`] = `"<div></div>"`;
